src/modifiers/containing_scope.py

```diff
@@ -8,7 +7,9 @@
 tag: user.cursorless
 """
 
-containing_scope_type_map = {
+
+mod.list("cursorless_containing_scope_type", desc="Supported containing scope types")
```

Comment:
I'll rename this one to
```suggestion
mod.list("cursorless_scope_type", desc="Supported scope types")
```

as we'll use these for other things later